url.bs

```diff
@@ -1431,13 +1431,11 @@ if all of the following are true:
 <ol>
  <li><p>Let <var>path</var> be <var>url</var>'s <a for=url>path</a>.
 
- <li><p>If <var>path</var> <a for=list>is empty</a>, then return.
-
  <li><p>If <var>url</var>'s <a for=url>scheme</a> is "<code>file</code>", <var>path</var>'s
  <a for=list>size</a> is 1, and <var>path</var>[0] is a <a>normalized Windows drive letter</a>, then
  return.
 
- <li><p><a for=list>Remove</a> <var>path</var>'s last item.
+ <li><p><a for=list>Remove</a> <var>path</var>'s last item, if any.
```

Comment:
@annevk I believe this is more concise compared to the previous definition while maintaining the precision, but happy to revert if you disagree.